src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHelpers.cs

```diff
@@ -113,11 +114,61 @@ internal static bool IsAllNullBytes(Span<byte> buffer)
             return true;
         }
 
-        // Returns a DateTimeOffset instance representing the number of seconds that have passed since the Unix Epoch.
-        internal static DateTimeOffset GetDateTimeFromSecondsSinceEpoch(double secondsSinceUnixEpoch)
+        // Converts the specified number of seconds that have passed since the Unix Epoch to a DateTimeOffset.
+        internal static DateTimeOffset GetDateTimeOffsetFromSecondsSinceEpoch(long secondsSinceUnixEpoch) =>
+            new DateTimeOffset((secondsSinceUnixEpoch * TimeSpan.TicksPerSecond) + DateTime.UnixEpoch.Ticks, TimeSpan.Zero);
+
+        // Converts the specified number of seconds that have passed since the Unix Epoch to a DateTimeOffset.
+        internal static DateTimeOffset GetDateTimeOffsetFromSecondsSinceEpoch(double secondsSinceUnixEpoch) =>
+            new DateTimeOffset((long)(secondsSinceUnixEpoch * TimeSpan.TicksPerSecond) + DateTime.UnixEpoch.Ticks, TimeSpan.Zero);
+
+        // Converts the specified DateTimeOffset to the number of seconds that have passed since the Unix Epoch.
+        internal static double GetSecondsSinceEpochFromDateTimeOffset(DateTimeOffset dateTimeOffset) =>
+            ((double)(dateTimeOffset.UtcDateTime - DateTime.UnixEpoch).Ticks) / TimeSpan.TicksPerSecond;
+
+        // If the specified fieldName is found in the provided dictionary and it is a valid double number, returns true and sets the value in 'dateTimeOffset'.
+        internal static bool TryGetDateTimeOffsetFromTimestampString(Dictionary<string, string>? dict, string fieldName, out DateTimeOffset dateTimeOffset)
+        {
+            dateTimeOffset = default;
+            if (dict != null &&
+                dict.TryGetValue(fieldName, out string? value) &&
+                double.TryParse(value, NumberStyles.Any, CultureInfo.InvariantCulture, out double secondsSinceEpoch))
+            {
+                dateTimeOffset = GetDateTimeOffsetFromSecondsSinceEpoch(secondsSinceEpoch);
+                return true;
+            }
+            return false;
+        }
+
+        // Converts the specified DateTimeOffset to the string representation of seconds since the Unix Epoch.
+        internal static string GetTimestampStringFromDateTimeOffset(DateTimeOffset timestamp)
+        {
+            double secondsSinceEpoch = GetSecondsSinceEpochFromDateTimeOffset(timestamp);
+            return secondsSinceEpoch.ToString("F9", CultureInfo.InvariantCulture); // 6 decimals, no commas
```

Comment:
The comment appears out of date now.

Reply:
Ah, right. If you don't mind, since it's just a comment, I can edit it in the next PR built on top of this one: https://github.com/dotnet/runtime/pull/70869 

I'd like to avoid restarting the CI just for a comment.